projects/metallb/metallb/helm/patches/0001-packages-patches.patch

```diff
@@ -93,6 +95,316 @@ index ef8824fe..be8cdd96 100644
  
  
  # A chart can be either an 'application' or a 'library' chart.
+diff --git a/charts/metallb/charts/frr-k8s-0.0.14.tgz b/charts/metallb/charts/frr-k8s-0.0.14.tgz
```

Comment:
Do you have steps to document this?

Reply:
Steps are same for creating the patch we are just removing the zip for frr-k8s in chart directory and updating the patch